src/os/layer/preset/presetmenubutton.js

```diff
@@ -382,15 +540,18 @@ const directive = () => ({
   controllerAs: 'ctrl',
   template: `
 <div class="btn-group o-add-data-button" ng-right-click="ctrl.openMenu()">
-  <button type="button" class="btn btn-primary col-auto" ng-click="ctrl.notifyApplyPreset()"
-    title="Apply the layer style preset">
-    <i class="fa fa-check"></i>
+  <button type="button" class="btn btn-primary col-auto" 
+      ng-click="ctrl.notifyApplyPreset()"
+      title="Apply the layer style preset">
+    <i class='fa fa-check'></i>
```

Comment:
Nit: single quotes in an HTML template. I thought the HTML linter disallowed this...